src/Authentication/Services/SystemUserService.cs

```diff
@@ -531,7 +532,7 @@ private async Task<Result<SystemUser>> InsertNewSystemUser(
                 return Problem.SystemUser_FailedToCreate;
             }
 
-            if (regSystem.Rights is not null && regSystem.Rights.Count > 0 && delegationCheckFinalResult is not null && delegationCheckFinalResult.CanDelegate)
+            if (newSystemUser.UserType == SystemUserType.Standard && regSystem.Rights is not null && regSystem.Rights.Count > 0 && delegationCheckFinalResult is not null && delegationCheckFinalResult.CanDelegate)
```

Comment:
can we have one bool that says can or not as the check is now going quite complex on a single line how about a method that says yes/no and use that here